docs/usage/config-presets.md

```diff
@@ -168,6 +168,10 @@ To host your preset config on GitLab:
 
 Note: Unlike npmjs-hosted presets, GitLab-hosted ones can contain only one config.
 
+## Local presets
+
+Renovate also supports local presets, i.e. presets that are hosted on the same platform as the target repository. This is especially helpful in self-hosted scenarios where public presets cannot be used. Local presets are only supported on GitHub and GitLab. Local presets are referenced via the `local>` prefix, e.g. `local>owner/name`. Renovate will determine the current platform and look up the preset from there.
```

Comment:
@rarkins  Why not just use `>` ? eg `>owner/name`. 🤔

Reply:
Actually this triggered my memory that we could perhaps use *no* prefix as discussed in #3019. @fgreinacher what do you think about that? According to someone who tested it, a preset `a/b` today will attempt to look up npm package `a%2fb` which can never exist. i.e. `a/b` is an "available" and perhaps intuitive namespace to use? i.e. "if the preset name contains one or more `/` and does not start with `@`, then assume it is a local repository"

Reply:
Not a strong opinion, but I'd prefer being explicit here

Reply:
i prefer shorter configs 😅

Reply:
@rarkins Your decision 👨‍✈

Reply:
@fgreinacher in that case let's do both :)

`a/b` and `local>a/b` being equivalent. Can you update this PR?

Reply:
Sure, will do :)

Reply:
Done with fb04453728bc9fbbf4e8f9119f36a81217075748